main-application/comms-processing_tasks.c

```diff
@@ -0,0 +1,72 @@
+/*  comms-processing_tasks.c
+
+    Created by Minh Phong Bill Truong on April 28th 2020
+ */
+#include "FreeRTOS.h"
+#include "task.h"
+#include "semphr.h"
+
+#include <stdio.h>
+#include "comms-processing_tasks.h"
+
+//==============================================================================
+//                                   GLOBALS
+//==============================================================================
```

Comment:
See following comments.
I would usually call this PRIVATE GLOBALS since you'd usually put globals you'd only want available to _this_ file here.

Reply:
Ack